main.go

```diff
@@ -56,61 +56,63 @@ func (l *LocalSSH) PrepareForEval(req *proto.PrepareForEvalRequest) (*proto.Prep
 	return &proto.PrepareForEvalResponse{}, nil
 }
 
-func (l *LocalSSH) Eval(request *proto.EvalRequest) (*proto.EvalResponse, error) {
```

Comment:
Once more, same as above. Accept the request, and read individual values from it. That makes changes backward compatible.